src/main/webapp/app/entities/participation-submission/participation-submission.component.html

```diff
@@ -0,0 +1,50 @@
+<h1>Submissions for Participation with id {{ participationId }}</h1>
+<br />
+<div class="table-responsive" *ngIf="submissions">
+    <ngx-datatable
```

Comment:
I cannot stress enough how cool it is that you took your time to integrate the data table here!
Very good work :+1: